go/tools/dokanclean/main.go

```diff
@@ -80,8 +80,10 @@ func removeKeybaseStartupShortcuts() {
 // If not just listing, execute each uninstaller we find and merge return codes.
 func findDokanUninstall(list bool, wow64 bool) (result int) {
 	var access uint32 = registry.ENUMERATE_SUB_KEYS | registry.QUERY_VALUE
+	// Assume this is build 32 bit, so we need this flag to see 64 but registry
```

Comment:
We spoke about it on Tuesday --- is there any way to mark dokan installs as belonging to keybase and the then only remove those that we installed?  If it's not in this build, can you make a JIRA story for it so we don't forget?  The sooner it's in the better.